tests/Unit/Evolution/Systems/CurvedScalarWave/Worldtube/ElementActions/Test_SendToWorldtube.cpp

```diff
@@ -196,7 +196,6 @@ SPECTRE_TEST_CASE("Unit.CurvedScalarWave.Worldtube.SendToWorldtube", "[Unit]") {
         {element_faces_grid_coords, dummy_time_step_id});
 
     ActionTesting::set_phase(make_not_null(&runner), Parallel::Phase::Testing);
-
```

Comment:
You're probably going to just drop the relevant commit, but make sure you don't end up with any changes to this file.